api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>I think it is more illustrative if we can add a diagonal line to the plot</p><p>plt.plot([min(ym_df['yield']), max(ym_df['yield'])], [min(ym_df['yield']), max(ym_df['yield'])])</p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
done